src/DiscreteFilter.cpp

```diff
@@ -0,0 +1,314 @@
+/*
+ * DiscreteFilter.cpp
+ *
+ *  Created on: 19/09/2014
+ *      Author: rafaelsaback
+ */
+
+#include "DiscreteFilter.hpp"
+
+namespace base {
```

Comment:
It's a bit confusing because FitPolynom and MatchTemplate both use 'numeric' namespace, while Histogram, PlaneFitting and Stats use 'base' namespace. So, independently of what namespace we use, we wouldn't have consistency inside the library anyway.

I could update the namespace of all to 'numeric', so it would be more consistent, but this would break some codes that use this libraries.

Anyway, I'll leave the decision up to you guys. So far I'm considering it as 'base'.